### PR TITLE
refactors confirmdeletemodal component

### LINKED ISSUE #493: Refactor Sketches/components/ConfirmDeleteModal.js to a functional component w/ hooks
https://github.com/uclaacm/TeachLAFrontend/blob/master/src/components/Sketches/components/ConfirmDeleteModal.js

### PR DESCRIPTION
**Pull request by ldalton02**
closes #493

### CODE REVIEW

File: src/components/Sketches/components/ConfirmDeleteModal.js [hunk 1]
```diff
@@ -1,92 +1,92 @@
-import React from 'react';
+import { programUploadSuccess } from 'actions/userDataActions.js';
```
Comment (reginawang99) [inline src/components/Sketches/components/ConfirmDeleteModal.js:1]: nit: can we remove this line if we aren't using the import?

**Review by reginawang99 — CHANGES_REQUESTED** [review]
looks mostly good to me! much speed 🚗 🔥 

one tiny thing: lets extract the props! it makes it easier to see what we props are actually being used

**Comment by reginawang99** [comment]
last thing! Could u add in closes #493 in the description?

**Review by reginawang99 — APPROVED** [review]
Looks good to me!! 🥳🎉 congrats on first react pull request :)) exciting times hehe

@lumisphere902 any thoughts?

**Review by lumisphere902 — APPROVED** [review]
LGTM. Now that I'm looking at it though... neither of the state variable are actually used lol. I think error can be removed, but spinner could be pretty easily implemented. What do you think?

**Comment by reginawang99** [comment]
hm i dont know editor super well, but i would just leave it in, cuz it doesn't really hurt anything. 

I'm not sure if y'all's were planning on using it in the future or anything? You probably know best

**Pull request merged**